sdk/purview/purview-workflow-rest/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-10-16)
```

Comment:
Is the release date correct?

Reply:
Updated this date to align with what's in merged PRs for SDKs in other languages.